scss/civicrm/event/_detail.scss

```diff
@@ -0,0 +1,180 @@
+#crm-container {
+  .crm-actions-ribbon {
+    margin-bottom: 20px;
+  }
+
+  .crm-form-block.crm-form-block>.crm-submit-buttons,
+  .crm-form-block .form-item>.crm-submit-buttons {
+    border-top: 0;
+    margin-top: 0 !important;
+  }
+
+  #newLocation {
+    padding: 0 20px 20px 15px;
+
+    #crm-submit-buttons {
+      padding-left: 5px;
+    }
+
+    .crm-event-is_show_location br {
+      display: none;
+    }
+
+    h3 {
+      @include section-title;
+      margin: 0 -20px 20px -15px;
+    }
+  }
+
+  .crm-event-manage-location-form-block > table {
+    margin-left: 15px;
+
+    .status {
+      padding-left: 0;
+    }
+  }
+
+  #priceSet .message {
+    margin-right: 15px;
+  }
+
+  #contributionType .description {
+    padding-right: 15px;
+  }
+
+  fieldset#discount,
+  fieldset#map-field {
+    padding-left: 20px;
+    padding-right: 20px;
+
+    > legend {
+      @include section-title;
+      display: block;
+      width: 100%;
+      margin-left: -20px;
+    }
+
+    table {
+      min-width: 100%;
+    }
+
+    tr {
+      background-color: transparent;
+      border-top: 1px solid $select2-inner-background;
+      display: block;
+      margin: 0 -20px;
+      padding: 0px 20px 0px 20px;
+    }
+
+    td {
+      border: 0;
+      display: inline-block;
+      padding: 5px 10px 9px;
+      width: 260px;
+    }
+
+    .dateplugin {
+      margin-top: 5px;
+    }
+
+    .columnheader {
+      display: block;
+      background-color: $gray-lighter;
+      border-bottom: 1px solid $crm-grayblue-dark;
+      border-left: 0;
+      border-radius: 0;
+      border-right: 0;
+      margin: 20px -20px 0;
+      padding: 6px 20px 4px 20px;
+
+      br {
+        display: none;
+      }
+
+      td {
+        color: $datepicker-gray-dark;
+        font-size: 13px;
+      }
+
+      label {
+        margin-left: 5px;
+      }
+    }
+  }
+
+  fieldset#discount {
+    .columnheader td:first-child,
+    .form-item td:first-child {
+      padding-left: 2px;
+      width: 20px;
+    }
+
+    .crm-form-submit {
+      margin: 10px 20px 0;
+    }
+  }
+
+  #isDiscount {
+    padding-left: 20px;
+    padding-right: 20px;
+
+    .crm-event-manage-fee-form-block-is_discount td {
+      width: auto;
+    }
+  }
+
+  .crm-collapsible {
+    &:not(.collapsed) .collapsible-title:before {
+      content: '\f107';
```

Comment:
@boriszaydlin sorry just found that we can use variables here as well from font-awesome